ReleaseHistory.md

```diff
@@ -4,6 +4,7 @@
   `System.Diagnostics.Debug` 4.3.0,
   `System.IO.FileSystem.Primitives` 4.3.0,
   `System.Text.Encoding.Extensions` 4.3.0.
+* NEW: Expose `MultithreadedAnalyzeCommandBase.BuildDisabledSkimmersSet`, a utility function which extracts a disabled skimmer set from a TContext.
```

Comment:
>TContext [](http://example.com/codeflow?start=138&length=8)

wrap this (and all named things, generally) in back ticks. Looks better in markdown. FCOR. :) :) :)